### ISSUE TITLE
Prometheus addon

### ISSUE THREAD

**Issue by p-hash** [hassio-addons/repository-beta#47]
# Problem/Motivation

Currently [prometheus addon repository](https://github.com/hassio-addons/addon-prometheus) is archived.
Most active fork is [here](https://github.com/lupino3/addon-prometheus), maintained by @lupino3 
I'd like to install the forked addon and see if its working or not, but there is no way to do it
I've tried adding `https://github.com/lupino3/addon-prometheus` as addon repo, but got an error that it is not a valid addon repository

## Expected behavior

Users able to test prometheus addon

## Actual behavior

There is no way to install prometheus addon right now, even for the testing purposes

## Steps to reproduce

Try to install Lupino3's [addon-prometheus](https://github.com/lupino3/addon-prometheus) to your HomeAssistant OS installation

## Proposed changes

Lupino3's [addon-prometheus](https://github.com/lupino3/addon-prometheus) is to be included to this [repository-beta](https://github.com/hassio-addons/repository-beta)

# TLDR: 
There is no way to install prometheus addon right now, even for the testing purposes. Can we include it in repository-beta?

**Comment by frenck** [comment]
Hi there @p-hash 👋 

It seems like you are in the wrong place. If you have question about installing their add-on, you should reach out to them :)

../Frenck
